### PR TITLE
Use index based keys with prefixed ids per terminal

### PR DESCRIPTION
**Pull request by sabarasaba**
Fixes: https://github.com/rohanchandra/react-terminal-component/issues/3

### CODE REVIEW

File: src/ReactTerminal.js [hunk 1]
```diff
@@ -122,6 +123,7 @@ class Terminal extends Component {
 };
 
 Terminal.propTypes = {
+  terminalId: PropTypes.number,
```
Comment (rohanchandra) [inline src/ReactTerminal.js]: Should be of type `string`: `PropTypes.string`.

File: src/OutputList.js [hunk 2]
```diff
@@ -27,7 +25,8 @@ const OutputList = ({ outputRenderers, outputs, ...outputRendererProps }) => (
 
 OutputList.propTypes = {
   outputRenderers: PropTypes.object.isRequired,
-  outputs: PropTypes.object.isRequired
+  outputs: PropTypes.object.isRequired,
+  terminalId: PropTypes.number,
```
Comment (rohanchandra) [inline src/OutputList.js]: Should be of type `string` not `number`: (use `PropTypes.string`)

**Review by rohanchandra — CHANGES_REQUESTED** [review]
Thanks for the fix! Just needs a small change to the `PropTypes` since a string ID is being used.

File: src/ReactTerminal.js (hunk 1 above)
Comment (sabarasaba) [reply src/ReactTerminal.js]: Good catch, should be fixed now :dancer:

**Review by rohanchandra — APPROVED** [review]

**Comment by rohanchandra** [comment]
Thanks again, I've merged your changes in

**Pull request merged**